ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form-tab-container.view.js

```diff
@@ -28,15 +28,15 @@ module.exports = Marionette.LayoutView.extend({
     collectionView: '.collection',
   },
   initialize: function() {
-    this.resultFormCollection = ResultForm.getResultCollection()
+    this.resultFormCollection = ResultFormCollection
     this.listenTo(
       this.resultFormCollection,
       'change:doneLoading',
       this.handleLoadingSpinner
     )
   },
   onRender: function() {
-    this.resultFormCollection.addResultForms()
+    ResultFormCollection.initialize()
```

Comment:
Usually we don't call initialize directly, it acts more like a constructor so this looks a bit odd.

Reply:
I thought it was odd as well because this `initialize` method body is empty. However, I was following the [steps taken](https://github.com/codice/ddf/blob/c8e6a44c933975c7b30e9489faa58fc211d8334d/ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form-tab-container.view.js#L38) in `search-form-tab-container.view.js`. I could remove both if you'd prefer?

Reply:
I'm not sure what to do here just thought I would mention it.